conans/client/graph/graph_builder.py

```diff
@@ -215,7 +219,7 @@ def _resolve_recipe(self, ref, graph_lock):
         return new_ref, dep_conanfile, recipe_status, remote
 
     @staticmethod
-    def _resolved_system_tool(node, require, profile_build, profile_host):
+    def _resolved_system_tool(node, require, profile_build, profile_host, resolve_prereleases):
```

Comment:
Maybe it would be more idiomatic to remove the ``staticmethod`` and use internally directly ``self._resolve_prereleases``, but I am fine with keeping it static and making evident what is passed as argument.